src/util/inconsistencies.py

```diff
@@ -80,7 +83,17 @@ def break_recursive_loops(cr, model, field, name_field="name"):
         name=SQL(get_value_or_en_translation(cr, table, name_field)),
     )
     cr.execute(update_query, [tuple(ids)])
-    bad_data = cr.fetchall()
+    n_updates = cr.rowcount
+    if n_updates / total_records > 0.1:
```

Comment:
Don't warn on small tables.
```suggestion
    if total_records > 100 and n_updates / total_records > 0.1:
```

Reply:
Can't we just log always? This should have been the default IMO. One line in the logs is cheap. This will also help us understand how widespread the loops are in some models.

Reply:
> Can't we just log always? This should have been the default IMO. One line in the logs is cheap. This will also help us understand how widespread the loops are in some models.

I also prefer this approach. We should log what happens in this util.